portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,24 @@
+@component-name = "portal"
+definition {
+
+	property testray.main.component.name = "Navigation";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	@description = "Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
+	@priority = 5
+	test AssertConfigurationSectionPresentInApplicationsMenu {
+		Click(locator1 = "MyPath#APPLICATIONS_MENU");
+
+		Click(locator1 = "MyPath#CONTROL_PANEL");
+
+		AssertElementPresent(locator1 = "MyPath#PANEL_CONFIGURATION_SECTION");
+	}
```

Comment:
Hey, I believe the aim of the test is to validate if the string `CONFIGURATION` is present (like you mentioned in the description of this test). 
The lines I highlighted for you do not assert if the string is present. What you need to do is:
1. Create a new XPath related to the string CONFIGURATION
2. Assert if the text is present

You can update your xpath to be more simple to validate this text:
1. See that the CONFIGURATION string is an HTML element of type `<h2></h2> `
2. See that this element has a unique `@id`, every id in an HTML document is unique
3. You can create your xpath using the h2 element and the id attribute

And then you can use a function to validate the CONFIGURATION text, there are a lot of functions to validate a text like `AssertTextEquals`

Please try this approach and see if it works, thank you!

Reply:
![Screenshot](https://github.com/manuferreira/liferay-portal/assets/90215790/c2ce1018-526d-4e20-94e2-711b80f8ecc9)

The description of the test says to assert that the CONFIGURATION section is showing, that's why I put the whole element instead of just the String. 

Thanks for the tips, by the way. Should I still make the changes you requested?

Reply:
Yes, I saw it and I agree this is a little misleading but let's follow the "Validando uma string/texto" and validate the text CONFIGURATION :slightly_smiling_face: 
but this does not mean that what you did is wrong, your code is correct but let's try this other approach. Thank you!

Reply:
Okay! :+1: